src/Diagnostics.RuntimeHost/Extensions/StorageServiceExtensions.cs

```diff
@@ -29,6 +29,10 @@ public static IServiceCollection AddDiagEntitiesStorageService(this IServiceColl
             {
                 services.AddSingleton<IStorageService, NullableStorageService>();
             }
+            else
+            {
+                services.AddSingleton<IStorageService, NullableStorageService>();
+            }
```

Comment:
You can just live with the else and remove the extra if statement 

`if (configuration.IsAzureChinaCloud() || configuration.IsAzureUSGovernment())`

Reply:
Hey @michimune , when is it will we set `CloudDomain` as `UnitTest`? Are we doing this somewhere in an actual unit test?

Reply:
I checked again and found it in AzureStorageTests.cs. Although the test code is currently commented out, I think it still makes sense to keep the UnitTest logic.
@puneetg1983 the reason I just added the else statement was to keep the intension of the original code. But I will add a comment to this logic so it makes more sense.

Reply:
Comments are added.